DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Reports/ReportsController.cs

```diff
@@ -81,5 +82,41 @@ public IEnumerable<ActivityDataRowModel> GetGraphData()
         // {
         //     return View();
         // }
+
+        [HttpGet]
+        [Route("Download")]
+        public IActionResult DownloadUsageData(
+            int? jobGroupId,
+            int? courseCategoryId,
+            int? customisationId,
+            string startDate,
+            string endDate,
+            ReportInterval reportInterval
+        )
+        {
+            var parsedStartDate = DateTime.Parse(startDate);
+            var parsedEndDate = DateTime.Parse(endDate);

```

Comment:
We probably want to do some validation on date times and ensure that:
* Start date is not before the earliest activityLog
* End date is after start date and not in the future

We could also use DateTime rather than string for automatic binding if we do validation.

Can we return 404 if that's not the case?